components/registration/registration-controller.js

```diff
@@ -10,11 +10,17 @@ import {
 import {enableAutoTransferFromNaerkontakt} from "../../ks_patches/custom_override_flags";
 import {
     INDEKSERING_PROGRAM_ID,
-    NAERKONTAKT_PROGRAM_ID, PROFIL_FNR, PROFIL_FNR_AS_WELL,
-    PROFIL_NASJONALT_FELLES_HJELPENUMMER
+    NAERKONTAKT_PROGRAM_ID,
+    PROFIL_FNR,
+    PROFIL_FNR_AS_WELL,

```

Comment:
Should we rename these two so they reflect which tracked entity type they belong to?

Reply:
Done!